lib/elastic_record/index/documents.rb

```
@@ -65,7 +65,7 @@ def index_record(record, index_name: alias_name)
       def update_record(record, index_name: alias_name)
         unless disabled
           update_document(
-            record.send(record.class.primary_key),
+            record.try(:id),
```

Should we raise an error on `update_record` if no ID is provided?

---

Ah, this should fail on the `update_document` call anway